src/dot-components/Wishlist/wishlist.css

```diff
@@ -0,0 +1,56 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,300;0,400;0,500;0,900;1,300&display=swap');
+
+body{
+    font-family: "Montserrat", sans-serif;
+}
```

Comment:
importing fonts in root is better will save the time and also you will not have to repeat.

Reply:
That will work across the whole project? Okay will change this.